rusk/tests/services/transfer.rs

```diff
@@ -185,7 +185,8 @@ pub async fn wallet() -> Result<()> {
     assert_ne!(original_root, new_root, "Root should have changed");
 
     // Revert the state
-    rusk.revert().expect("Reverting should succeed");
+    let base_root = rusk.base_root();
+    rusk.revert(base_root).expect("Reverting should succeed");
```

Comment:
It might be worth defining a function to revert to base root, or making `revert` reverting to `base_root` if invoked with empty argument.
Just saying because this is done in several points.

Reply:
Done